pages/auth.vue

```diff
@@ -0,0 +1,46 @@
+<script setup>
+    // login() only returns a code challenge. This script turns the code challenge into an auth-token
+    const params = new URLSearchParams(window.location.search);
+    if (params.has("error")) {
+        console.log(params.get("error"));
+        let errorUrl = new URL("/error").searchParams.append("redirect-uri", "/login");
+        navigateTo(errorUrl);
+    }
+    
+    const form = {
+        grant_type: "authorization_code",
+        code: params.code,
+        redirect_uri: window.location.href, // required, but not used for redirect
```

Comment:
warum speicherst du die redirect_uri in login.vue, wenn du hier eh was anders nimmst
ps. href klappt nicht, weil da mit die search params sind
pps. und ersetz mal 'not used for redirect' => 'no actual redirect'